src/home-page/components/CategoryCard.tsx

```diff
@@ -7,26 +7,28 @@ const CategoryCardContainer = styled(Card)`
   display: flex;
   align-items: center;
 
- :hover {
-   opacity: 0.8;
- }   
+  :hover {
+    opacity: 0.8;
+  }
 `;
 
 const CardHeaderContainer = styled(CardHeader)`
- width: 100%;
+  width: 100%;
 `;
 
-const AvatarContainer = styled(Avatar)<{backgroundColor: string}>`
- margin-right: 1em;
- background-color: ${props => props.backgroundColor};
+const AvatarContainer = styled(Avatar)<{ background: string }>`
+  margin-right: 1em;
+  background-color: ${({ background, theme }) => theme.palette.professions[background].main};
```

Comment:
We're accessing `background` prop here with bracket notation which is convenient, but unfortunately also means we are no longer guarded by TypeScript type checking. What I mean is that if we pass a wrong string to `AvatarContainer`, e.g. we make a typo and pass `fontend`, we may break our code.
What we could do to is use a type union: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html. To do this, we need to declare the following:

`type ProfessionKey = 'frontend' | 'backend' | 'uxui' | ... ;`

And add an interface for props:
```
interface Props = {
   background: ProfessionKey;
}
```